src/WrappedMToken.sol

```diff
@@ -438,6 +438,9 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
 
         emit Transfer(sender_, recipient_, amount_);
 
+        // Return early if sender and recipient are the same account.
```

Comment:
```suggestion
 _revertIfInvalidRecipient(recipient_);
 
 emit Transfer(sender_, recipient_, amount_);
 
 // Return early if sender and recipient are the same account.
if (sender_ == recipient_) return;

// Claims for both the sender and recipient are required before transferring since add an subtract functions
// assume accounts' balances are up-to-date with the current index.
_claim(sender_, currentIndex_);
_claim(recipient_, currentIndex_);
```